client/src/components/Comparison/YourProducts/YourProducts.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Card from '../Card/Card.jsx';
+import yourProducts from './YourProducts.module.css';
+
+class YourProducts extends React.Component {
+  render() {
+    return (
+      <div className="your-products">
+        <div>
+          RELATED PRODUCTS
+        </div>
+        <div className={yourProducts.main}>
+          <div className={yourProducts['card-container']}>
+            <Card />
```

Comment:
same - lots of different `<Card />` placeholders here

Reply:
Put them as placeholders to hold the actual products later